hack/test-end-to-end-scenario.sh

```diff
@@ -18,6 +18,7 @@ function wait_for_app() {
 	echo "[INFO] Waiting for app in namespace $1"
 	echo "[INFO] Waiting for database pod to start"
 	wait_for_command "oc get -n $1 pods -l name=database | grep -i Running" $((60*TIME_SEC))
+	oc deploy database --logs -n $1
```

Comment:
The only case the pod will be deleted here is if the deployment is successful. `deploy --logs` won't error out, instead it will print a success message. So this won't flake (at least not because the deployment was successful).